server/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -984,8 +985,8 @@ public String toString() {
             org.elasticsearch.tasks.TaskCancelledException::new, 146, Version.V_5_1_1),
         SHARD_LOCK_OBTAIN_FAILED_EXCEPTION(org.elasticsearch.env.ShardLockObtainFailedException.class,
                                            org.elasticsearch.env.ShardLockObtainFailedException::new, 147, Version.V_5_0_2),
-        UNKNOWN_NAMED_OBJECT_EXCEPTION(org.elasticsearch.common.xcontent.NamedXContentRegistry.UnknownNamedObjectException.class,
-                org.elasticsearch.common.xcontent.NamedXContentRegistry.UnknownNamedObjectException::new, 148, Version.V_5_2_0),
+        UNKNOWN_NAMED_OBJECT_EXCEPTION(UnknownNamedObjectException.class,
+                UnknownNamedObjectException::new, 148, Version.V_5_2_0),
```

Comment:
Would you make this consistent with the rest of the code and use the fully-qualified name here (instead of adding an import)?